frontend/packages/dev-console/src/components/import/builder/BuilderSection.tsx

```diff
@@ -16,7 +16,7 @@ const BuilderSection: React.FC<ImageSectionProps> = ({ image, builderImages }) =
   }
 
   return (
-    <FormSection title="Builder" divider>
+    <FormSection title="Builder" fullWidth>
```

Comment:
This is causing the `BuilderImageTagSelector` to also be full width which isn't correct. The dropdown width should be constrained. You'll need to return a fragment with multiple `FormSection`s.